documentation/modules/oauth/proc-oauth-authentication-broker-config.adoc

```diff
@@ -121,12 +121,18 @@ Depending on how you apply OAuth 2.0 authentication, and the type of authorizati
     fallbackUserNamePrefix: client-account- <3>
     validTokenType: bearer <4>
     userInfoEndpointUri: __https://OAUTH-SERVER-ADDRESS/auth/realms/external/protocol/openid-connect/userinfo__ <5>
+    enablePlain: true <6>
+    tokenEndpointUri: __https://OAUTH-SERVER-ADDRESS/auth/realms/external/protocol/openid-connect/token__ <7>
+    enableOAuthBearer: false <8>
```

Comment:
For easier readability, should the `enablePlain` and `enableOAuthBearer` be next to each other?

Reply:
I put `tokenEndpointUri` after `enablePlain` because it's relevant only when `enablePlain` is `true`, so it made sense to me to put it there. But we can also put it after `enableOAuthBearer` so the other two are next to each other.

Reply:
> But we can also put it after enableOAuthBearer so the other two are next to each other.

I guess that makes sense then.

Reply:
What about flipping the order like this:
```
enableOAuthBearer: false
enablePlain: true
tokenEndpointUri: https://...
```

Reply:
Yeah, that works for me perfectly fine.